### ISSUE TITLE
HealthLine anti-GDPR

### ISSUE THREAD

**Issue by iam-py-test** [DandelionSprout/adfilt#922]
## Which entry/entries are you submitting?
`healthline.com`

## Which things do they block, hide, or unbreak?
When visiting their website (with an EU IP?) they give you a cookie consent (normal).
![image](https://github.com/DandelionSprout/adfilt/assets/84232764/ca624820-d45c-4d17-9d48-80bb5c4d22ef)
If you manage settings, you have an option to opt-out of everything
![image](https://github.com/DandelionSprout/adfilt/assets/84232764/147fa644-caac-4980-803b-d350a335044e)
If you disallow all, they redirect you to this weird version which only has three articles (anon.healthline.com)
![image](https://github.com/DandelionSprout/adfilt/assets/84232764/a90cd389-228c-41f9-86c4-dec12e212914)
Example URL: `https://www.healthline.com/health/pregnancy/can-a-girl-get-pregnant-by-another-girl` (don't ask)
TLDR: they want you to either consent tracking, or not use their website
I'm not 100% sure this fits in the GDPR list, though...

## Which of my lists are you submitting it to?
GDPR 451 List of Mostly American Websites Who Hate Europe and Privacy Rights

## Which adblocker(s) and version did you use when writing and testing the entries?

- [X] uBlock Origin
- [ ] AdGuard (Paid desktop version)
- [ ] AdGuard (Gratis browser versions that aren't Manifest V3 nor Safari)
- [ ] AdBlock (Firefox or Manifest V2)
- [ ] Adblock Plus (Firefox or Manifest V2)
- [ ] AdGuard Home
- [ ] Blokada
- [ ] I Don't Care About Cookies (The extension)
- [ ] AdNauseam

Other(s):

Adblocker version(s):

**Comment by iam-py-test** [comment]
Also `https://www.medicalnewstoday.com/articles/lesbian-pregnancy-options` (owned by HealthLine?)

**Comment by ArtexJay** [comment]
> https://www.medicalnewstoday.com/

Can confirm MedicalNewsToday.com owned by the UK branch of HealthLine Media.

**Comment by iam-py-test** [comment]
Thanks

**Comment by DandelionSprout** [comment]
I'll look into that site later this week, likely no earlier than Thursday.

**Comment by iam-py-test** [comment]
Ok, it's not urgent.
Thanks

**Comment by DandelionSprout** [comment]
My conclusion may sound a bit, but I've discovered that since I Don't Care About Cookies (An extension/list that I'm a big fan of) simply removes the cookie prompts entirely even for EEC users, it at least won't end up in GDPR 451 List.

And as none of my other lists would be suited for blocking `healthline.com` in, I'll acquit that site for now.
